test/fixtures/output/connect/cluster/update-error.golden

```diff
@@ -0,0 +1 @@
+Error: unable to read configuration file "test/fixtures/input/connect/update-config-malformed.json": offsets are not allowed in configuration file for update
```

Comment:
```suggestion
Error: unable to read configuration file "test/fixtures/input/connect/update-config-malformed.json": offsets are not allowed in updated configuration file
```

Reply:
May be  i can do `offsets are not allowed in configuration file for update command`

Reply:
> offsets are not allowed in configuration file for `confluent connect cluster update`